mmv1/third_party/terraform/website/docs/d/secret_manager_secrets.html.markdown

```diff
@@ -0,0 +1,132 @@
+---
+subcategory: "Secret Manager"
+description: |-
+  List the Secret Manager Secrets.
+---
+
+# google\_secret\_manager\_secrets
+
+Use this data source to list the Secret Manager Secrets
+
+## Example Usage 
+
+
+```hcl
+data "google_secret_manager_secrets" "secrets" {
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - (optional) The ID of the project.
+
+* `filter` - (optional) A filter expression that
```

Comment:
could this all be modified to not have so many line breaks? Thanks!

Reply:
I've updated the field description to be consistent with the [API docs](https://cloud.google.com/secret-manager/docs/reference/rest/v1/projects.secrets/list#query-parameters)